doc/install/mne_python.rst

```diff
@@ -79,13 +81,16 @@ conda to ``/home/user/anaconda3``)::
     (probably at or near the beginning), but the ``command not found`` error
     suggests that it is missing.
 
-    On Linux or OSX, the installer should have put something
+    On Linux or macOS, the installer should have put something
     like the following in your ``~/.bashrc`` or ``~/.bash_profile``
     (or somewhere else if you are using a non-bash terminal):
```

Comment:
The default shell on macOS is now `zsh` (which `conda` readily supports). So I'm wondering if we should replace `bash` with a more generic term in this manual.

Reply:
> Also, the latest installers modify only `.bash_profile` for me, not `.bashrc`

Strange, I just set up a new linux machine ~6 weeks ago and it put the `<<< conda initialize <<<` block in `~/.bashrc`.  `~/.profile` is untouched, and `~/.bash_profile` doesn't exist.